packages/common/test/pipes/date_pipe_spec.ts

```diff
@@ -71,6 +71,12 @@ import {JitReflector} from '@angular/platform-browser-dynamic/src/compiler_refle
     describe('transform', () => {
       it('should use "mediumDate" as the default format',
          () => expect(pipe.transform('2017-01-11T10:14:39+0000')).toEqual('Jan 11, 2017'));
+
+      it('should return first week if some dates fall in previous year but belong to next year accordint to ISO 8601 format', () => {
```

Comment:
Drive-by suggestion:

```suggestion
      it('should return first week if some dates fall in previous year but belong to next year according to ISO 8601 format', () => {
```

Reply:
@petebacondarwin thanks made the change